tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -24,3 +35,9 @@ Verify User Can Spawn A Notebook
     Wait For RHODS Dashboard To Load
     Launch JupyterHub Spawner From Dashboard
     Spawn Notebook With Arguments    image=s2i-minimal-notebook    size=Default
+
+Update Access Groups In RHODS Groups Config
```

Comment:
In ODS.robot there is a keyword `Set Default Access Groups Settings` that does the same

Reply:
Updated